docs/gettingstarted/installation-macos.rst

```diff
@@ -18,7 +18,18 @@ PostgreSQL
 
 Saleor needs PostgreSQL version 9.4 or above to work. Get the macOS installer from the `PostgreSQL download page <https://www.postgresql.org/download/macosx/>`_.
 
-Make sure you keep track of the password you set for the administration account during installation.
+Then, create a new database and user. Make sure you keep track of the password you set for the administration account during installation. 
+
+.. code-block:: console
+
+ $ psql postgres
+
+and from the Postgres prompt:
+
+.. code-block:: psql
+
+ # CREATE ROLE saleor WITH LOGIN PASSWORD 'saleor' SUPERUSER;
+ # CREATE DATABASE saleor;
```

Comment:
I would put the two instructions into two separated blocks for better understanding and structure:
1. Create the database;
2. Create the superuser with a password.